dice_ml/explainer_interfaces/dice_genetic.py

```diff
@@ -480,7 +480,7 @@ def find_counterfactuals(self, query_instance, desired_range, desired_class,
             predictions = self.predict_fn_scores(population[i])[0]
             if self.is_cf_valid(predictions):
                 self.final_cfs.append(population[i])
-                if not isinstance(predictions, float) and len(predictions) > 1:
+                if not isinstance(predictions, float) and not isinstance(predictions, np.float32) and len(predictions) > 1:
```

Comment:
what is the problem with float predictions? Is it because model is classifier. in that case, can you add a comment mentioning the reason?

also what about float64? does this cover all possible float/double types?